kubespawner/spawner.py

```diff
@@ -800,6 +802,93 @@ def _hub_connect_port_default(self):
         """
         )
 
+    form_template = Unicode(
```

Comment:
Hmm, I wonder if we can make this be a jinja2 template instead? That gives users a lot more control than just python formatstrings. It would let us have just one 'form_template' rather than have a form_template and an inputs template.

Reply:
does jinja included in kubespawner? All other template I see use string.format templating

Reply:
jinja2 is included in JupyterHub. I think the other templates use string.format because they are fairly simple. I'd like jinja2 for this one because it sets up HTML, and string.format might easily allow XSS and other issues. All the html templating in JupyterHub is jinja2.

Reply:
So I have not tested it yet, but I have switched to jinja rendered + docstring